lib/ts/string_view.h

```diff
@@ -241,9 +240,6 @@ template <typename _Type, typename _CharTraits> class basic_string_view
   // std::string constructor
   constexpr basic_string_view(std::string const &rhs) noexcept : m_data(rhs.data()), m_size(rhs.size()) {}
 
-  // ats_scoped_str constructor
-  constexpr basic_string_view(ats_scoped_str const &rhs) noexcept : m_data(rhs.get()), m_size(traits_type::length(rhs.get())) {}
```

Comment:
I am sure there is no need. I added that because I thought there is no way of constructing it with a function inside ats_scoped_str. Just found out use get() inside ats_scoped_source is totally fine :)